### DISCUSSION TITLE
Need help about certificate p12 lost

### DISCUSSION THREAD

**Discussion by jeuxdaily** [airsdk/Adobe-Runtime-Support#3457] [General]
hello
I have old google play account
I lost certificate p12  creating with adobe animate cc
how can I recovery my certificate with password

**Comment by marchbold** [comment]
As far as I'm aware you pretty much have to move on, certificates are practically impossible to recover. 

I believe your only option would be to contact Google about changing the signature of the application and using a new certificate. This change has issues but is likely the only option if you need to recover and update that application.

**Comment by jeuxdaily** [comment]
hi
I need command linle to the encrypted private key with "pepk-src.jar"

![11](https://github.com/user-attachments/assets/701d7f24-c017-41e8-8474-98494cd6c2a6)

**Reply by marchbold** [reply]
Have a look here: https://github.com/airsdk/Adobe-Runtime-Support/discussions/3405

**Reply by jeuxdaily** [reply]
thanks man
but I less my certificate with pass
how can I do

**Reply by marchbold** [reply]
As I mentioned above if you have lost your certificate you will need to contact google about changing it. The upload certificate is something different to your application certificate.

**Reply by jeuxdaily** [reply]
I need command line working with pepl-src.jar 
When I put the following code 
java -jar C:\elias\pepk-src.jar --keystore=C:\elias\certf.p12 --alias=1 --output=fffff.zip --encryptionkey=gmcakeencrypt --include-cert

on CMD

I get this error

no main manifest attribute in pepk-src.jar

**Reply by ajwfrost** [reply]
Looks like that's a wrong Jar file, it seems Google have an incorrect link on their website.
See https://stackoverflow.com/questions/44719351/googles-pepk-src-not-working

**Reply by jeuxdaily** [reply]
thanks
I've seen this before.
I tried with this code but after uploading the file I find the following message
the private key doesn't match the current app certificate

**Comment by jeuxdaily** [comment]
hi
so I find a command line to show content p12 files

keytool -list -v -keystore certf.p12 -storepass name -storetype PKCS12

so 
Now I need to modify it if possible
cause  I have a stupid idea to find certificate name with password
thanks
